src/Polly.Contrib.DuplicateRequestCollapser.RedisDistributedLock/Polly.Contrib.DuplicateRequestCollapser.RedisDistributedLock.csproj

```
@@ -36,6 +37,30 @@
   <ItemGroup>
     <PackageReference Include="Polly" Version="7.2.0" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+    <PackageReference Include="Polly.Contrib.DuplicateRequestCollapser">
+      <Version>0.2.0</Version>
+    </PackageReference>
```

```suggestion
    <PackageReference Include="Polly.Contrib.DuplicateRequestCollapser" Version="0.2.0"</PackageReference>
```

(and similarly for all other PackageReference Version sub-elements in file)

Only because [this line in the build](https://github.com/Polly-Contrib/Polly.Contrib.DuplicateRequestCollapser.RedisDistributedLock/blob/20feac53c0fa7e38be5abf0b3141a1825d93f19f/build.cake#L182) will unwantedly replace the `<Version>0.2.0</Version>` of the PackageReference if it is a sub-element rather than property; when only intending to target [this line](https://github.com/Polly-Contrib/Polly.Contrib.DuplicateRequestCollapser.RedisDistributedLock/blob/20feac53c0fa7e38be5abf0b3141a1825d93f19f/src/Polly.Contrib.DuplicateRequestCollapser.RedisDistributedLock/Polly.Contrib.DuplicateRequestCollapser.RedisDistributedLock.csproj#L7).

(If you fancy writing a multi-line regex in the `cake.build` to handle that case better, feel free!)